src/components/10-atoms/radio/index.js

```diff
@@ -159,29 +159,38 @@ class AXARadio extends NoShadowDOM {
 
     // controlledness is a React-only notion
     this.state.isControlled = isControlled && isReact;
+    const inputElement = html`
+      <input
+        id="${refId}"
+        class="a-radio__input"
+        type="radio"
+        name="${name}"
+        ?checked="${checked}"
+        value="${value}"
```

Comment:
For better grouping, I would move `value` 1 line up.

Reply:
✅